services/account/apps/api/src/api.module.ts

```diff
@@ -43,28 +42,17 @@ import { ApiService, AccountsService, HandlesService, DelegationService, KeysSer
       // disable throwing uncaughtException if an error event is emitted and it has no listeners
       ignoreErrors: false,
     }),
-    RedisModule.forRootAsync(
-      {
-        imports: [ConfigModule, EventEmitterModule],
-        useFactory: (configService: ConfigService, eventEmitter: EventEmitter2) => ({
-          config: [
-            {
-              url: configService.redisUrl.toString(),
-              keyPrefix: configService.cacheKeyPrefix,
-              maxRetriesPerRequest: null,
-              onClientCreated(client) {
-                redisEventsToEventEmitter(client, eventEmitter);
-              },
-              readyLog: false,
-              errorLog: false,
-            },
-          ],
-        }),
-        inject: [ConfigService, EventEmitter2],
-      },
-      true, // isGlobal
-    ),
-    QueueModule.forRoot(),
+    CacheModule.forRootAsync({
+      useFactory: (configService: ConfigService) => [
+        {
+          url: configService.redisUrl.toString(),
+          keyPrefix: configService.cacheKeyPrefix,
+          maxRetriesPerRequest: null,
+        },
+      ],
+      inject: [ConfigService],
+    }),
+    QueueModule,
```

Comment:
1:1 swap, `RedisModule` is now initialized inside `CacheModule`. This pattern across all Gateway apps & services.